<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenCfger</class>
 <widget class="QFrame" name="ScreenCfger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>130</height>
   </rect>
  </property>
  <widget class="QLabel" name="labelCardModel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Card Model:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelScreenWidth">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Screen Width:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelScreenHeight">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Screen Height:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelModuleColor">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Module Color:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCommunication">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Communication:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPort">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>labelCardModel</zorder>
  <zorder>label_1</zorder>
  <zorder></zorder>
  <zorder>labelScreenWidth</zorder>
  <zorder>label_2</zorder>
  <zorder>labelScreenHeight</zorder>
  <zorder>label_3</zorder>
  <zorder>labelModuleColor</zorder>
  <zorder>label_4</zorder>
  <zorder>labelCommunication</zorder>
  <zorder>label_5</zorder>
  <zorder>labelPort</zorder>
  <zorder>label_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
